examples/llm/benchmarks/README.md

```diff
@@ -43,21 +43,20 @@ docker compose -f deploy/docker_compose.yml up -d
 
 ## Disaggregated Single Node Benchmarking
 
-In the following steps we compare Dynamo disaggregated vLLM single node performance to
-[native vLLM Aggregated Baseline](#vllm-aggregated-baseline-benchmarking). These were chosen to optimize
+*One H100 80GB x8 node is required for this setup.*
+
+In the following setup we compare Dynamo disaggregated vLLM performance to
+[native vLLM Aggregated Baseline](#vllm-aggregated-baseline-benchmarking) on a single node. These were chosen to optimize
```

Comment:
Can we have a small section on the configurations being run? tp and dp, how many prefill workers, how many decode workers - I might have missed it - but we should spell that out here in the beginning

Reply:
Yes, the section is added.